src/components/Kv/KvExpandableQuestion.vue

```diff
@@ -1,41 +1,43 @@
 <template>
-	<div class="expandable-faq">
-		<button class="expandable-faq__header"
+	<div>
```

Comment:
might not be a bad idea to put `class="kv-tailwind"` on this outer div in case the component is used in other places that don't have a kv-tailwind parent

Reply:
oh I see, elsewhere you've wrapped it in a kv-tailwind div. 👍

Reply:
I don't think KvExpandableQuestion  should be used in outside of KvFrequentlyAskedQuestions..